docs/Language.md

```diff
@@ -0,0 +1,64 @@
+The following guidelines for writing copy are intended to create a consistent style and voice for the intelliHR product.
+
+## Language
+* **Keep text short and simple** to make text accessible to people of all backgrounds
+    * **Do**: This person doesn't have a job record
```

Comment:
What's our stance on doesn't vs does not?

Reply:
Ah i forgot to copy the part about contractions. Google's conversation API thing prescribes that contractions should always be used because `cannot` and `do not` sound much harsher than `can't` and `don't`

Reply:
makes sense!